.github/workflows/wheels.yml

```diff
@@ -124,11 +123,22 @@ jobs:
 #        if: ${{ runner.os == 'Windows' && env.IS_32_BIT == 'true' }}
 
       - name: Build wheels
-        uses: pypa/cibuildwheel@v2.12.1
+        uses: pypa/cibuildwheel@v2.14.1
```

Comment:
I think it'd be better to move to `2.15` here, that's the first release with proper Python 3.12 support without use of `CIBW_PRERELEASE_PYTHONS`. @andyfaff WDYT?